xml/hwbp_registers.xml

```diff
@@ -987,7 +996,16 @@
             If the H extension is not supported, the only legal values are 0 and 4.
         </field>
 
-        <field name="0" bits="22:18" access="R" reset="0" />
+        <field name="0" bits="22:20" access="R" reset="0" />
+        
+        <field name="sbytemask" bits="19:18" access="WARL" reset="0">
+            0=compare, 1=don't compare byte, i.e. byte is masked out
+            bit 18 = lower byte compare of svalue (bits [7:0])
+            bit 19 = upper byte compare of svalue (bits [15:8])
```

Comment:
We use capitalization and punctuation in field descriptions. When mentioning the behavior of values we use : instead of = (see e.g. https://github.com/riscv/riscv-debug-spec/blob/master/xml/dm_registers.xml#L12).

I don't think we have any examples yet of where each bit inside a field has a different meaning. The closest is hawindow, where we say "I.e. bit 0 refers to hart $\RdmHawindowsel * 32$, while bit 31 refers to hart $\RdmHawindowsel * 32 + 31$." (https://github.com/riscv/riscv-debug-spec/blob/master/xml/dm_registers.xml#L347).

Putting that all together, I think we want language like:
When the least-significant bit of this field is 1, it causes bits 7:0 in the comparison to be ignored when \FcsrTextraThirtytwoSselect=1. When the next most significant bit of this field is 1, causes bits 15:8 to be ignored in the comparison when \FcsrTextraThirtytwoSselect=1, and so on.